internal/sdk/internal/hooks/registration.go

```diff
@@ -12,7 +12,4 @@ func initHooks(h *Hooks) {
 	// Add hooks by calling h.register{SDKInit/BeforeRequest/AfterSuccess/AfterError}Hook
 	// with an instance of a hook that implements that specific Hook interface
 	// Hooks are registered per SDK instance, and are valid for the lifetime of the SDK instance
-	requestablePermissionsHook := &RequestablePermissionsHook{}
-
-	h.registerBeforeRequestHook(requestablePermissionsHook)
```

Comment:
add back?

Reply:
Yes, I guess the regeneration blows this away :(

Reply:
Re-added